parsing/parsetree.mli

```diff
@@ -188,214 +204,242 @@ and object_field_desc =
   | Otag of label loc * core_type
   | Oinherit of core_type
 
-(* Patterns *)
+(** {2 Patterns} *)
 
 and pattern =
     {
      ppat_desc: pattern_desc;
      ppat_loc: Location.t;
      ppat_loc_stack: location_stack;
-     ppat_attributes: attributes; (* ... [@id1] [@id2] *)
+     ppat_attributes: attributes;  (** [... [\@id1] [\@id2]] *)
     }
 
 and pattern_desc =
   | Ppat_any
-        (* _ *)
+        (** The pattern [_]. *)
   | Ppat_var of string loc
-        (* x *)
+        (** The variable pattern such as [x] *)
```

Comment:
`The variable pattern` ⇒ `A variable pattern` since there are more than one variable patterns?